.github/workflows/deploy.yml

```diff
@@ -301,10 +227,10 @@ jobs:
         continue-on-error: true
         env:
           BASE_URL: ${{ needs.deploy.outputs.application_endpoint }}
-          CYPRESS_STATE_USER_EMAIL: ${{ secrets.CYPRESS_STATE_USER_EMAIL }}
-          CYPRESS_STATE_USER_PASSWORD: ${{ secrets.CYPRESS_STATE_USER_PASSWORD }}
-          CYPRESS_ADMIN_USER_EMAIL: ${{ secrets.CYPRESS_ADMIN_USER_EMAIL }}
-          CYPRESS_ADMIN_USER_PASSWORD: ${{ secrets.CYPRESS_ADMIN_USER_PASSWORD }}
+          E2E_STATE_USER_EMAIL: ${{ secrets.E2E_STATE_USER_EMAIL }}
```

Comment:
I'm assuming you made the changes in the Github admin secrets for this repo

Reply:
I missed that.  Good catch!  Thank you!